pkg/util/test/util.go

```diff
@@ -2862,12 +2862,12 @@ func ValidateTelemetry(pxImageList map[string]string, cluster *corev1.StorageClu
 	logrus.Info("Validate Telemetry components")
 	logrus.Info("Check PX and PX Operator versions to determine which telemetry to validate against..")
 	pxVersion := GetPortworxVersion(cluster)
-	fmt.Printf("PX Version: %v", pxVersion)
+	logrus.Infof("PX Version: [%s]", pxVersion.String())
```

Comment:
nit: pxVersion should be good without String()

Reply:
Both `GetPortworxVersion()` and `GetPxOperatorVersion()` return `*version.Version`, I figured I'd do same for both